### PR TITLE
Add OSF

### LINKED ISSUE #3651: Addition of Icon: OSF
**Name:** OSF
**Website:** https://osf.io/
**Alexa rank:** 52,644
  

**Official resources for icon and color:**  https://osf.io/

### PR DESCRIPTION
**Pull request by einbandi**
OSF (short for "Open Science Framework") is a collection of services for collaborating on and sharing research projects. OSF is provided by the Center for Open Sciences.

osf-preview

**Issue:** closes #3651

**Similarweb rank:** 31,915

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description

- I manually vectorized the icon, using a gap of 0.5 px between the colored faces.
- I added "Open Science Framework" as an alias. The provider of the OSF services, the "Center for Open Sciences", uses the same logo. I added this title ("Center for Open Sciences") under `"dup"`, but I weasn't quite sure if this was necessary.
- There are multiple logo files across different OSF and COS repos and websites, but I could not find any SVGs, apart from a terrible one (see #3651). The highest-quality PNG that I found is [this one](https://github.com/CenterForOpenScience/osf.io/blob/de170682924278eba1db9d6e1c50166bf43700e0/website/static/img/circle_logo.png) from the `osf.io` repo, but the location of the circles is different across different versions. I aligned and sized the shapes to the best of my knowledge.
- I also picked the hex color ("2CB9F1") from the above PNG.

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
> Hello, from the future

[Hello from the past](https://github.com/simple-icons/simple-icons/issues/3651#issuecomment-755259215)!

**Comment by PeterShaggyNoble** [comment]
Gave this a try myself and came up with the following; can someone throw an eye over it compared to the `source` PNG before I go any further with it?

```
M 12.087891 0A 2.9844999 2.9844999 0 0 0 9.1015625 2.984375A 2.9844999 2.9844999 0 0 0 9.6269531 4.6640625C 10.000477 3.6541102 10.973659 2.9296874 12.109375 2.9296875C 13.230143 2.9296876 14.192028 3.6358228 14.576172 4.625A 2.9844999 2.9844999 0 0 0 15.072266 2.984375A 2.9844999 2.9844999 0 0 0 12.087891 0zM 5.6503906 2.6660156A 2.9844999 2.9844999 0 0 0 2.6660156 5.6503906A 2.9844999 2.9844999 0 0 0 5.6503906 8.6347656A 2.9844999 2.9844999 0 0 0 8.6347656 5.6503906A 2.9844999 2.9844999 0 0 0 5.6503906 2.6660156zM 18.349609 2.6660156A 2.9844999 2.9844999 0 0 0 15.365234 5.6503906A 2.9844999 2.9844999 0 0 0 18.349609 8.6347656A 2.9844999 2.9844999 0 0 0 21.335938 5.6503906A 2.9844999 2.9844999 0 0 0 18.349609 2.6660156zM 12.109375 3.4296875A 2.1465001 2.1465001 0 0 0 9.9628906 5.5761719A 2.1465001 2.1465001 0 0 0 12.109375 7.7226562A 2.1465001 2.1465001 0 0 0 14.255859 5.5761719A 2.1465001 2.1465001 0 0 0 12.109375 3.4296875zM 9.1132812 6.0410156C 8.9357313 7.6192852 7.6952542 8.8846175 6.1289062 9.1015625A 2.1465001 2.1465001 0 0 0 7.4921875 9.59375A 2.1465001 2.1465001 0 0 0 9.6386719 7.4472656A 2.1465001 2.1465001 0 0 0 9.1132812 6.0410156zM 14.894531 6.0878906A 2.1465001 2.1465001 0 0 0 14.40625 7.4472656A 2.1465001 2.1465001 0 0 0 16.552734 9.59375A 2.1465001 2.1465001 0 0 0 17.908203 9.1054688C 16.340151 8.9054631 15.092822 7.6564635 14.894531 6.0878906zM 2.984375 9.015625A 2.9844999 2.9844999 0 0 0 0 12A 2.9844999 2.9844999 0 0 0 2.984375 14.984375A 2.9844999 2.9844999 0 0 0 4.6425781 14.478516C 3.6396199 14.101356 2.921875 13.130688 2.921875 12C 2.921875 10.868667 3.6406752 9.8981804 4.6445312 9.5214844A 2.9844999 2.9844999 0 0 0 2.984375 9.015625zM 21.015625 9.015625A 2.9844999 2.9844999 0 0 0 19.341797 9.5351562C 20.339561 9.9149761 21.052734 10.882815 21.052734 12.009766C 21.052734 13.130534 20.346599 14.09242 19.357422 14.476562A 2.9844999 2.9844999 0 0 0 21.015625 14.984375A 2.9844999 2.9844999 0 0 0 24 12A 2.9844999 2.9844999 0 0 0 21.015625 9.015625zM 5.5683594 9.8535156A 2.1465001 2.1465001 0 0 0 3.421875 12A 2.1465001 2.1465001 0 0 0 5.5683594 14.146484A 2.1465001 2.1465001 0 0 0 7.7148438 12A 2.1465001 2.1465001 0 0 0 5.5683594 9.8535156zM 18.40625 9.8632812A 2.1465001 2.1465001 0 0 0 16.259766 12.009766A 2.1465001 2.1465001 0 0 0 18.40625 14.15625A 2.1465001 2.1465001 0 0 0 20.552734 12.009766A 2.1465001 2.1465001 0 0 0 18.40625 9.8632812zM 7.4921875 14.376953A 2.1465001 2.1465001 0 0 0 6.09375 14.894531C 7.667474 15.096154 8.919189 16.354463 9.109375 17.931641A 2.1465001 2.1465001 0 0 0 9.6386719 16.523438A 2.1465001 2.1465001 0 0 0 7.4921875 14.376953zM 16.552734 14.376953A 2.1465001 2.1465001 0 0 0 14.40625 16.523438A 2.1465001 2.1465001 0 0 0 14.898438 17.884766C 15.109383 16.316648 16.368953 15.072381 17.945312 14.888672A 2.1465001 2.1465001 0 0 0 16.552734 14.376953zM 5.6503906 15.365234A 2.9844999 2.9844999 0 0 0 2.6660156 18.349609A 2.9844999 2.9844999 0 0 0 5.6503906 21.335938A 2.9844999 2.9844999 0 0 0 8.6347656 18.349609A 2.9844999 2.9844999 0 0 0 5.6503906 15.365234zM 18.349609 15.365234A 2.9844999 2.9844999 0 0 0 15.365234 18.349609A 2.9844999 2.9844999 0 0 0 18.349609 21.335938A 2.9844999 2.9844999 0 0 0 21.335938 18.349609A 2.9844999 2.9844999 0 0 0 18.349609 15.365234zM 12 16.273438A 2.1465001 2.1465001 0 0 0 9.8535156 18.419922A 2.1465001 2.1465001 0 0 0 12 20.566406A 2.1465001 2.1465001 0 0 0 14.146484 18.419922A 2.1465001 2.1465001 0 0 0 12 16.273438zM 14.267578 19.777344C 13.803554 20.54754 12.959983 21.066406 12 21.066406C 11.094244 21.066406 10.290099 20.605243 9.8125 19.90625A 2.7344999 2.6345 0 0 0 9.3515625 21.365234A 2.7344999 2.6345 0 0 0 12.085938 24A 2.7344999 2.6345 0 0 0 14.820312 21.365234A 2.7344999 2.6345 0 0 0 14.267578 19.777344z
```

File: icons/osf.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....18 0 0 0-.156-.176 2.18 2.18 0 0 0-1.568-.639zM5.613 15.65a2.725 2.725 0 0 0-1.924 4.656 2.718 2.718 0 0 0 1.928.8 2.71 2.71 0 0 0 1.926-.8 2.72 2.72 0 0 0 0-3.853 2.728 2.728 0 0 0-1.93-.803zm12.772 0a2.724 2.724 0 0 0-1.928.803 2.72 2.72 0 0 0 0 3.854c.533.533 1.23.798 1.926.798s1.394-.265 1.928-.798a2.725 2.725 0 0 0-1.926-4.657zm-6.381.842a1.93 1.93 0 0 0-1.936 1.928c0 .108.015.211.032.314v.002c.016.1.043.194.074.287.006.02.012.04.02.059.03.084.069.164.11.242.014.026.027.052.042.076a1.9 1.9 0 0 0 .476.536c.05.038.1.077.155.11.052.034.106.061.162.089.05.025.099.053.152.074.069.027.142.045.215.064.042.012.082.027.125.036a1.948 1.948 0 0 0 .748 0c.038-.008.072-.022.11-.032.076-.02.152-.04.224-.068.058-.023.112-.054.168-.082.048-.025.097-.05.142-.078.055-.034.104-.073.155-.111.04-.032.08-.062.117-.096a1.907 1.907 0 0 0 .363-.447l.031-.06c.045-.082.087-.165.12-.255.006-.018.01-.038.015-.057.031-.093.058-.19.074-.29a1.93 1.93 0 0 0 .032-.313 1.921 1.921 0 0 0-1.926-1.928zM9.69 19.146a2.975 2.975 0 0 0-.666 1.88A2.975 2.975 0 0 0 11.97 24a2.975 2.975 0 0 0 .03 0 2.975 2.975 0 0 0 2.975-2.975 2.975 2.975 0 0 0-.666-1.877 2.424 2.424 0 0 1-2.305 1.7c-1.084 0-2-.72-2.313-1.702z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/osf.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>OSF</title><path d="M12.088 0a2.984 2.984 0 0 0-2.986 2.984 2.984 2.984 0 0 0 .525 1.68 2.654 2.654 0 0 1 2.482-1.734c1.121 0 2.083.706 2.467 1.695a2.984 2.984 0 0 0 .496-1.64A2.984 2.984 0 0 0 12.088 0zM5.65 2.666A2.984 2.984 0 0 0 2.666 5.65 2.984 2.984 0 0 0 5.65 8.635 2.984 2.984 0 0 0 8.635 5.65 2.984 2.984 0 0 0 5.65 2.666zm12.7 0a2.984 2.984 0 0 0-2.985 2.984 2.984 2.984 0 0 0 2.985 2.985 2.984 2.984 0 0 0 2.986-2.985 2.984 2.984 0 0 0-2.986-2.984zm-6.24.764a2.147 2.147 0 0 0-2.147 2.146 2.147 2.147 0 0 0 2.146 2.147 2.147 2.147 0 0 0 2.147-2.147 2.147 2.147 0 0 0-2.147-2.146zM9.112 6.04a3.497 3.497 0 0 1-2.984 3.06 2.147 2.147 0 0 0 1.363.493A2.147 2.147 0 0 0 9.64 7.447a2.147 2.147 0 0 0-.526-1.406zm5.782.047a2.147 2.147 0 0 0-.489 1.36 2.147 2.147 0 0 0 2.147 2.146 2.147 2.147 0 0 0 1.355-.489 3.494 3.494 0 0 1-3.013-3.017zM2.985 9.016A2.984 2.984 0 0 0 0 12a2.984 2.984 0 0 0 2.984 2.984 2.984 2.984 0 0 0 1.659-.505A2.655 2.655 0 0 1 2.922 12c0-1.131.719-2.102 1.723-2.479a2.984 2.984 0 0 0-1.66-.505zm18.03 0a2.984 2.984 0 0 0-1.673.52 2.655 2.655 0 0 1 1.71 2.474c0 1.12-.705 2.082-1.695 2.467a2.984 2.984 0 0 0 1.659.507A2.984 2.984 0 0 0 24 12a2.984 2.984 0 0 0-2.984-2.984zm-15.447.838A2.147 2.147 0 0 0 3.422 12a2.147 2.147 0 0 0 2.146 2.146A2.147 2.147 0 0 0 7.715 12a2.147 2.147 0 0 0-2.147-2.146zm12.838.01a2.147 2.147 0 0 0-2.146 2.146 2.147 2.147 0 0 0 2.146 2.146 2.147 2.147 0 0 0 2.147-2.146 2.147 2.147 0 0 0-2.147-2.147zM7.492 14.376a2.147 2.147 0 0 0-1.398.518 3.496 3.496 0 0 1 3.015 3.037 2.147 2.147 0 0 0 .53-1.409 2.147 2.147 0 0 0-2.147-2.146zm9.06 0a2.147 2.147 0 0 0-2.146 2.146 2.147 2.147 0 0 0 .492 1.362 3.495 3.495 0 0 1 3.047-2.996 2.147 2.147 0 0 0-1.392-.512zm-10.902.988a2.984 2.984 0 0 0-2.984 2.985 2.984 2.984 0 0 0 2.984 2.986 2.984 2.984 0 0 0 2.985-2.986 2.984 2.984 0 0 0-2.985-2.985zm12.7 0a2.984 2.984 0 0 0-2.985 2.985 2.984 2.984 0 0 0 2.985 2.986 2.984 2.984 0 0 0 2.986-2.986 2.984 2.984 0 0 0-2.986-2.985zm-6.35.908a2.147 2.147 0 0 0-2.146 2.147A2.147 2.147 0 0 0 12 20.566a2.147 2.147 0 0 0 2.146-2.146A2.147 2.147 0 0 0 12 16.273zm2.268 3.504A2.65 2.65 0 0 1 12 21.067a2.65 2.65 0 0 1-2.188-1.16 2.734 2.635 0 0 0-.46 1.458A2.734 2.635 0 0 0 12.086 24a2.734 2.635 0 0 0 2.734-2.635 2.734 2.635 0 0 0-.552-1.588z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]

**Review by adamrusted — APPROVED** [review]
@PeterShaggyNoble 👏🏻

**Pull request merged**